api/src/main/java/org/geysermc/geyser/api/event/bedrock/SessionLoadResourcePacksEvent.java

```diff
@@ -48,6 +49,13 @@ public SessionLoadResourcePacksEvent(@NonNull GeyserConnection connection) {
      */
     public abstract @NonNull List<ResourcePack> resourcePacks();
 
+    /**
+     * Gets an unmodifiable list of {@link ResourcePackCDNEntry}s that will be sent to the client.
+     *
+     * @return an unmodifiable list of resource pack CDN entries that will be sent to the client.
+     */
+    public abstract @NonNull List<ResourcePackCDNEntry> cdnEntries();
```

Comment:
Its not like we have a formal style guide, but if the javadoc body is the same as the `@return` tag, can we just have the return tag only?

Reply:
Makes sense, but this has been accepted before & i'd rather keep it consistent (or change all instances of that in api)

Reply:
RFC :P

and it exists elsewhere afaik